deku-c/deku-c-toolkit/src/contract.ts

```diff
@@ -152,6 +152,10 @@ export class Contract {
           return null;
         })
         .catch(console.error);
-    }, 2000);
+    }, 1000);
+  }
+
+  cancelOnNewState(): void {
+    if (this.fetchInterval) clearInterval(this.fetchInterval);
   }
```

Comment:
Could you add a doc to explain what this function does ?

Reply:
Done.